exporter/collector/integrationtest/testcases/conversion.go

```diff
@@ -166,11 +182,70 @@ func convertHistogram(h pmetric.Histogram) metricdata.Aggregation {
 			Sum:          pt.Sum(),
 			Bounds:       pt.ExplicitBounds().AsRaw(),
 			BucketCounts: pt.BucketCounts().AsRaw(),
+			Exemplars:    convertFloatExemplars(pt.Exemplars()),
+		}
+	}
+	return agg
+}
+
+func convertIntHistogram(h pmetric.Histogram) metricdata.Aggregation {
+	if h.DataPoints().Len() == 0 {
+		return nil
+	}
+	agg := metricdata.Histogram[int64]{
+		Temporality: convertTemporality(h.AggregationTemporality()),
+		DataPoints:  make([]metricdata.HistogramDataPoint[int64], h.DataPoints().Len()),
+	}
+	for i := 0; i < h.DataPoints().Len(); i++ {
+		pt := h.DataPoints().At(i)
+		agg.DataPoints[i] = metricdata.HistogramDataPoint[int64]{
+			Attributes:   attribute.NewSet(convertAttributes(pt.Attributes())...),
+			StartTime:    pt.StartTimestamp().AsTime(),
+			Time:         pt.Timestamp().AsTime(),
+			Count:        pt.Count(),
+			Sum:          int64(pt.Sum()),
+			Bounds:       pt.ExplicitBounds().AsRaw(),
+			BucketCounts: pt.BucketCounts().AsRaw(),
+			Exemplars:    convertIntExemplars(pt.Exemplars()),
 		}
 	}
 	return agg
 }
 
+func convertIntExemplars(es pmetric.ExemplarSlice) []metricdata.Exemplar[int64] {
+	exemplars := make([]metricdata.Exemplar[int64], es.Len())
+	for i := 0; i < es.Len(); i++ {
+		e := es.At(i)
+		traceID := e.TraceID()
+		spanID := e.SpanID()
```

Comment:
same nit comment as other spot about how these local vars are only used once

Reply:
I can't seem to do e.TraceID()[:]:
```
invalid operation: e.TraceID() (value of type pcommon.TraceID) (slice of unaddressable value) (compile)
```